tools/sgen/input/FluentUIStyle.yml

```diff
@@ -84,11 +84,11 @@ Typography:
   subheadline: "Font(subheadline)"
 
 Border:
-  radius: [ none: 0pt, small: 2pt, medium: 4pt, large: 8pt, xlarge: 12pt ]
+  radius: [ none: 0pt, small: 2pt, medium: 4pt, large: 8pt, xLarge: 12pt , circleSmall: 24pt, circleLarge: 26pt]
```

Comment:
Is `circleSmall` and `circleLarge` defined in Figma?

Reply:
We have a definition of circle as 50% the height, but I don't think that actually translates to design tokens very well.

Reply:
Alternatively, we can create a category for Persona sizes as they are used in ListCell + Avatar tokens to avoid any definition confusion in the future. It will probably increase the sgen size though and I'm not sure how beneficial the tradeoff would be

Reply:
Switched the implementation to use .clipShape(Capsule()) instead